_posts/ianmoore/2016-04-07-ian-moar-dictionary.md

```diff
@@ -0,0 +1,16 @@
+---
+layout: post
+author: touchwick
+title: "Ian's MOAR DICTIONARY STUFF Post"
+---
+This is exercise 1:
+
+<iframe src="https://trinket.io/embed/python3/d2a4bf04f5" width="100%" height="356" frameborder="0" marginwidth="0" marginheight="0" allowfullscreen></iframe>
```

Comment:
This trinket looks like a blank one.  Did you get the embed code from the Share menu?